lib/clients/reports/rates_and_costs_report.ex

```diff
@@ -0,0 +1,66 @@
+defmodule GlassFactoryApi.Clients.Reports.RatesAndCostsReport do
+  @moduledoc """
+  Defines the RatesAndCostsReport of a client report.
+  """
+  alias GlassFactoryApi.Clients.Reports.RatesAndCostsReport
+
+  @type t() :: %RatesAndCostsReport{
+          client_id: integer(),
+          project_id: integer(),
+          job_id: integer(),
+          activity_id: integer(),
+          user_id: integer(),
+          role_id: integer(),
+          date: String.t(),
+          time: integer(),
+          rate: integer(),
+          cost: integer()
+        }
+
+  defstruct [
+    :client_id,
+    :project_id,
+    :job_id,
+    :activity_id,
+    :user_id,
+    :role_id,
+    :date,
+    :time,
+    :rate,
+    :cost
+  ]
+
+  @doc """
+  Parse a map into a RatesAndCostsReport struct
+
+  ## Examples
+
+      iex> GlassFactoryApi.Clients.Reports.RatesAndCostsReport.to_struct(%{client_id: 1234, project_id: 12345...})
+      %RatesAndCostsReport{
+        client_id: 1234,
+        project_id: 12345,
+        job_id: nil,
+        activity_id: 123_456,
+        user_id: 222,
+        role_id: 1234,
+        date: "2018-06-19",
```

Comment:
I wonder if we should parse these Date values or just let them go through as strings.

Reply:
Yeah, me too my friend. Maybe a good approach to follow is to just implement the API calls as is and after handle each data as your real format, maybe we could do so when we start to develop the part of enrich api data, wdyt?

Reply:
Sounds good. Let's start with the simplest approach and map 1-to-1 on the types of the returned JSON then standardize stuff later 👍